setup.cfg

```diff
@@ -20,4 +21,4 @@ classifiers =
     Programming Language :: Python :: Implementation :: PyPy
 
 [options]
-python_requires = >=3.7
+python_requires = >=3.6
```

Comment:
please don't touch this or setup.cfg

Reply:
Oh sorry, that was not meant to get in this pr

Reply:
```suggestion
python_requires = >=3.7
```

Reply:
Other topic, but please keep in mind that most of the automotive world still depends on ubuntu18.04 which still uses 3.6.

Reply:
if the automotive world would like free support for end of lifed software they should be sponsoring